src/App.js

```diff
@@ -5,17 +5,33 @@ import data from './data/dummy.json';
 const App = () => {
   return (
     <div>
-      <ol>
-        {data.map((question) => {
-          return <div>
-            <li key={question.id}>{question.text}</li>
-            <ul>{question.answers.map((answer) => {
-              return <li key={answer.id}>{answer.text}</li>
-            })}
-            </ul>
-          </div>
-        })}
-      </ol>
+      <form>
+        <ol>
+          {data.map(question => {
+            return (
+              <div>
+                <li key={question.id}>{question.text}</li>
```

Comment:
Now that we are not using HTML lists, we can refactor the code. We are now using radio elements, and it is recommended to wrap radio inputs in `fieldset`s and include a `legend` heading within it. One of the big benefits of this is increased accessibility.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Reply:
Done :)